setup.py

```diff
@@ -510,7 +525,8 @@ def bundle_libzmq_extension(self):
 
             # And things like sockets come from libraries that must be named.
 
-            libzmq.libraries.extend(['rpcrt4', 'ws2_32', 'advapi32'])
+            libzmq.libraries.extend(['rpcrt4', 'ws2_32', 'advapi32', 'libsodium'])
+            libzmq.library_dirs.append('build/temp.win32-2.7/Release/buildutils')
```

Comment:
Fixed by copying similar code for `libzmq` in the top-level `bundled_settings` function.